src/app/shared/org-logo.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class OrgLogoService {
+  setDefault(img: any) {
+    // Check/set a flag to ensure the default logo is set only once, to avoid an infinite load loop if the default org image is not loadable
+    // IMPORTANT: BEFORE YOU CHANGE THE BELOW CODE, MAKE SURE YOU COMPLETELY UNDERSTAND IT!
```

Comment:
this seems a bit more alarming than normal, is that intentional?

Reply:
Yup, intentionally scary.  If the following code is incorrectly modified (that is, the "change" check is removed/broken) and the default image is not loadable for whatever reason, the browser will request the default image over and over again, hitting our server many times per second, until the page changes.  I can tone down the warning if you want. :)